src/symbol.ts

```diff
@@ -1,58 +1,54 @@
 import {Sprite, Texture} from "pixi.js";
 import {gsap} from "gsap";
-import {getRandomInt} from "./utils";
+import TweenVars = gsap.TweenVars;
 
 interface SlotSymbol {
     startBounce(): Promise<void>;
+
     moveOneSlot(): Promise<void>;
+
     endBounce(): Promise<void>;
+
     setSymbol(name: Texture): void;
 }
 
+interface GsapConfig {
```

Comment:
Consider using more abstract naming. This looks like some sort of "animation" config. While GSAP is a specific animations library. (the point here is that today GSAP is a way to go for animations in this project, but tomorrow it will be tween.js, and then something else. Therefore having "loose naming coupling" is preferred)